src/components/Footer.js

```diff
@@ -26,7 +26,15 @@ export const Footer = connect(({ authenticated, status, settings, user }) => ({
     cursorBack()
   }}>
     <li>
-
+      <span className="floatLeft">
+        <a className='increaseFont' onClick={() => {
+          if (scaleSize < 2) dispatch({ type: 'scaleSize', value: scaleSize + 0.1 })
```

Comment:
Please add three constants to `constants.js`:

- `MIN_FONT_SIZE`
- `MAX_FONT_SIZE`
- `FONT_SCALE_INCREMENT`

Reply:
Added in 60ff4d7